bbl_screen-patch/patches/printerui/qml/printer/SpeedAdjust.qml

```diff
@@ -0,0 +1,385 @@
+                            var innerY = centerY + (radius - 8) * Math.sin(angle);
+                            if (i % 10 === 0) {
+                                innerX = centerX + (radius - 15) * Math.cos(angle); //increase length of major ticks
+                                innerY = centerY + (radius - 15) * Math.sin(angle);
+                            }
+                            ctx.beginPath();
+                            ctx.moveTo(outerX, outerY);
+                            ctx.lineTo(innerX, innerY);
+                            var currentValue = dial.from + i;
+
+                            ctx.strokeStyle =  "#6B6B6A";
+                            ctx.lineWidth = 3;
+                            ctx.stroke();
+                            
+                            if (i % 10 === 0) {
+                                var labelRadius = radius - 30;
+                                var labelX = centerX + labelRadius * Math.cos(angle) - 10; 
+                                var labelY = centerY + labelRadius * Math.sin(angle) + 5;
+                                ctx.fillStyle = "white";
+                                ctx.fillText(currentValue, labelX, labelY);
+                            }
+                        }
+                    }
+                }
+            }
+
+            handle: Rectangle {
+                id: dialHandle 
+                width: 25
+                height: width
+                radius: width/2
+                color: dial.pressed ? Colors.gray_300 : Colors.gray_400
+                x: dialBackground.x + dialBackground.width / 2 - dialHandle.width / 2
+                y: dialBackground.y + dialBackground.height / 2 - dialHandle.height / 2
+                    transform: [
+                    Translate {
+                        y: -Math.min(dialBackground.width, dialBackground.height) * 0.5 + dialHandle.height / 2 + 3.5
+                    },
+                    Rotation {
+                        angle: dial.angle
+                        origin.x: dialHandle.width / 2
+                        origin.y: dialHandle.height / 2
+                    }
+                ]
+            }  
+    
+            Text {
+                id: valueText
+                text: Math.round(dial.value) + "%\n" + PrintManager.currentTask.printSpeedModeStrings[nearestLevel(targetSpeed)]
+                color: "white"
+                font.pixelSize: 36
+                horizontalAlignment: Text.AlignHCenter
+                anchors.centerIn: dial
+            }
+            
+        }
+        
+        Text { 
+            id: txtParams //displays the parameters that will be applied
+            property var speedParams: gcode.printSpeed(dial.value)
+            text:qsTr("Acceleration:\n%1\nFeed rate:\n%2\nTime:\n%3")
+                .arg(speedParams.accelerationMagnitude.toFixed(2))
+                .arg(speedParams.feedRate.toFixed(2))
+                .arg(speedParams.speedFraction.toFixed(1)) 
```

Comment:
Maybe the following might be more understandable?
```suggestion
            text:qsTr("Acceleration:\n%1%%\nFeed rate:\n%2%%\nTime:\n%3%%")
                .arg((speedParams.accelerationMagnitude*100.0).toFixed(0))
                .arg((speedParams.feedRate*100.0).toFixed(0))
                .arg((speedParams.speedFraction*100.0).toFixed(0)) 
```

Reply:
I think I'm also going to add a tooltip button in the upper right corner so that users don't have to hunt for documentation to figure out what these numbers mean